os/board/rtl8721csm/src/component/os/tizenrt/ethernetif_tizenrt.c

```diff
@@ -145,6 +145,42 @@ static void low_level_init(struct netif *netif)
  *       dropped because of memory failure (except for the TCP timers).
  */
 
+#ifdef CONFIG_NET_NETMGR
+
+err_t low_level_output(struct netdev *dev, uint8_t *data, uint16_t dlen)
+{
+	struct eth_drv_sg sg_list[MAX_ETH_DRV_SG];
+	int sg_len = 0;
+	struct pbuf *q;
+
+#if CONFIG_WLAN
```

Comment:
`#if` is used when it is defined as 0 or as 1. If you define it when you use, you should use `#ifdef`. Could you confirm this?

Reply:
Yes, I confirm #if is correct here.